README.rst

```diff
@@ -38,25 +38,29 @@ Quick start
-+===================================+============================================================================+
-| Track data                        | | ``$ git add train.py``                                                   |
-|                                   | | ``$ dvc add images.zip``                                                 |
-+-----------------------------------+----------------------------------------------------------------------------+
-| Connect code and data             | | ``$ dvc run -n prepare -d images.zip -o images/ unzip -q images.zip``    |
-|                                   | | ``$ dvc run -n train -d images/ -d train.py -o model.p python train.py`` |
-+-----------------------------------+----------------------------------------------------------------------------+
-| Make changes and reproduce        | | ``$ vi train.py``                                                        |
-|                                   | | ``$ dvc repro model.p.dvc``                                              |
-+-----------------------------------+----------------------------------------------------------------------------+
-| Share code                        | | ``$ git add .``                                                          |
-|                                   | | ``$ git commit -m 'The baseline model'``                                 |
-|                                   | | ``$ git push``                                                           |
-+-----------------------------------+----------------------------------------------------------------------------+
-| Share data and ML models          | | ``$ dvc remote add myremote -d s3://mybucket/image_cnn``                 |
-|                                   | | ``$ dvc push``                                                           |
-+-----------------------------------+----------------------------------------------------------------------------+
+
++-----------------------------------+----------------------------------------------------------------------------------+
+| Task                              | Terminal                                                                         |
++===================================+==================================================================================+
+| Track data                        | | ``$ git add train.py params.yaml``                                             |
+|                                   | | ``$ dvc add images.zip``                                                       |
++-----------------------------------+----------------------------------------------------------------------------------+
+| Connect code and data             | | ``$ dvc stage add -n prepare -d images.zip -o images/ unzip -q images.zip``    |
+|                                   | | ``$ dvc stage add -n train -d images/ -d train.py -o model.p python train.py`` |
```

Comment:
Do we need a `dvc exp run` or `dvc repro` call here?

Reply:
makes sense, yep

Reply:
actually it comes at the next step?